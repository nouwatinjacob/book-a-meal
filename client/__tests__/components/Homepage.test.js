import React from 'react';
import {
  shallow
} from 'enzyme';
import HomePage from '../../src/components/Homepage.jsx';

describe('HomePage component', () => {
  const setup = () => shallow(<HomePage />);

  it('should render HomePage component correctly', (done) => {
    const wrapper = setup();
    expect(wrapper).toMatchSnapshot();

    done();
  });
});