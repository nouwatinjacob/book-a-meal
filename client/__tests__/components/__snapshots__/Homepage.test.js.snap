// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage component should render HomePage component correctly 1`] = `
<div
  className="container"
>
  <div
    className="hero"
  >
    <div
      className="leading"
    >
      <div
        className="wrapper"
      >
        <h2
          className="leading-text"
        >
          Book A Meal is an online store for
          <br />
          ordering your favourite meals
        </h2>
        <button
          className="button warning"
          id="visit"
        >
          <Link
            replace={false}
            to="/menus"
          >
            Browse Menu
          </Link>
        </button>
      </div>
    </div>
  </div>
  <div
    className="wrapper"
  >
    <h3
      className="text-center"
    >
      How Book-A-Meal
      <br />
       Order works
    </h3>
    <div
      className="row"
    >
      <div
        className="c-medium-4"
        id="pd-0"
      >
        <div
          className="box"
        >
          <img
            alt="Menu Meals"
            className="article-img"
            src="https://res.cloudinary.com/sansaristic/image/upload/v1533140471/BookMeal/Default%20images/Meal-menu.png"
            style={
              Object {
                "width": "100%",
              }
            }
          />
          <div
            className="box-body"
          >
            <p
              className="text"
            >
              Browse menus and 
              <strong>
                your order
              </strong>
              in seconds without any delay
            </p>
          </div>
        </div>
      </div>
      <div
        className="c-medium-4"
        id="pd-0"
      >
        <div
          className="box"
        >
          <img
            alt="Fast Delivery"
            className="article-img"
            src="https://res.cloudinary.com/sansaristic/image/upload/v1533140467/BookMeal/Default%20images/fast-delivery.png"
            style={
              Object {
                "width": "100%",
              }
            }
          />
          <div
            className="box-body"
          >
            <p
              className="text"
            >
              <strong>
                Explore restaurants 
              </strong>
              that deliver to your doorstep
            </p>
          </div>
        </div>
      </div>
      <div
        className="c-medium-4"
        id="pd-0"
      >
        <div
          className="box"
        >
          <img
            alt="Real Time"
            src="https://res.cloudinary.com/sansaristic/image/upload/v1533140467/BookMeal/Default%20images/24-hrs.png"
            style={
              Object {
                "width": "100%",
              }
            }
          />
          <div
            className="box-body"
          >
            <p
              className="text"
            >
              Follow the status of your order with
              <strong>
                real-time alerts
              </strong>
            </p>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
